clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -5024,11 +5023,25 @@ bool SYCLIntegrationFooter::emit(raw_ostream &OS) {
   Policy.SuppressTypedefs = true;
   Policy.SuppressUnwrittenScope = true;
 
+  llvm::SmallVector<const VarDecl *> VisitedSpecConstants;
```

Comment:
Would it be better to use `set` instead of `vector` for quicker search? Or we don't expect a lot of items here and `vector` would be just good enough?